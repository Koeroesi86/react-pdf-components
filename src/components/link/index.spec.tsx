import React from "react";
import { renderToString } from "@react-pdf/renderer";
import { Wrapper } from "@test-helpers";
import Link from "./index";

describe("<Link/>", () => {
  it("should render", async () => {
    const result = await renderToString(
      <Wrapper>
        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
        <Link src="https://example.com"/>
      </Wrapper>
    );
    expect(result).toMatchSnapshot();
  });
});
