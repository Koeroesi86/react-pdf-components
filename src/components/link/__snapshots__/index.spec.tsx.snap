// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link/> should render 1`] = `
"%PDF-1.3
%����
7 0 obj
<<
/Type /Page
/Parent 1 0 R
/MediaBox [0 0 595.280029 841.890015]
/Contents 5 0 R
/Resources 6 0 R
>>
endobj
6 0 obj
<<
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI]
>>
endobj
5 0 obj
<<
/Length 71
>>
stream
1 0 0 -1 0 841.890015 cm
q
q
1 0 0 1 0 0 cm
q
q
1 0 0 1 0 0 cm
Q
Q
Q
Q

endstream
endobj
9 0 obj
(react-pdf)
endobj
10 0 obj
(react-pdf)
endobj
11 0 obj
(D:20210414183756Z)
endobj
8 0 obj
<<
/Producer 9 0 R
/Creator 10 0 R
/CreationDate 11 0 R
>>
endobj
3 0 obj
<<
/Type /Catalog
/Pages 1 0 R
/Names 2 0 R
/ViewerPreferences 4 0 R
>>
endobj
1 0 obj
<<
/Type /Pages
/Count 1
/Kids [7 0 R]
>>
endobj
2 0 obj
<<
/Dests <<
  /Names [
]
>>
>>
endobj
4 0 obj
<<
/DisplayDocTitle true
>>
endobj
xref
0 12
0000000000 65535 f 
0000000573 00000 n 
0000000630 00000 n 
0000000486 00000 n 
0000000677 00000 n 
0000000200 00000 n 
0000000133 00000 n 
0000000015 00000 n 
0000000412 00000 n 
0000000321 00000 n 
0000000348 00000 n 
0000000376 00000 n 
trailer
<<
/Size 12
/Root 3 0 R
/Info 8 0 R
>>
startxref
720
%%EOF
"
`;
